tests/test_dumper.py

```diff
@@ -50,23 +50,83 @@ class ClassFieldRenamed:
     yield ClassFieldRenamed
 
 
-@pytest.mark.parametrize("cls", cls_variant())
-def test_typed_fields(cls):
-    int_field = random.randrange(100)
-    str_field = "fhjldgfhjsdfgj"
-
-    res = dump(cls(int_field, str_field))
-    assert res == f"int_field: {int_field}\nstr_field: {str_field}\n"
-
-
-def test_plain_class():
-    class ThatClass:
-        int_field: int
-        str_field: str
-
-        def __init__(self, i_fld, s_fld):
-            self.int_field = i_fld
-            self.str_field = s_fld
-
-    res = dump(ThatClass(1, "2"))
-    assert res == "!!python/object:test_dumper.ThatClass\nint_field: 1\nstr_field: '2'\n"
+# @pytest.mark.parametrize("cls", cls_variant())
```

Comment:
Please uncomment these tests before finishing development

Reply:
Oh whoops! Shouldn't have committed that. I just commented those out to make debugging easier for myself